modules/translation/translation.go

```diff
@@ -25,17 +25,18 @@ type Locale interface {
 
 // LangType represents a lang type
 type LangType struct {
```

Comment:
Maybe a crazy idea here but can't we join this with type locale?

Reply:
It's feasible, and we could make a better locale struct.

In fact I had a plan to refactor all `{{TimeSince .StartTime $.i18n.Lang}}` to `{{i18nUtil.TimeSince .StartTime}}` to make code more simple and clear.